src/coreclr/tools/Common/Compiler/CompilerTypeSystemContext.cs

```diff
@@ -190,6 +190,11 @@ private EcmaModule AddModule(string filePath, string expectedSimpleName, bool us
             try
             {
                 PEReader peReader = OpenPEFile(filePath, out mappedViewAccessor);
+#if !READYTORUN
+                if (peReader.HasMetadata && (peReader.PEHeaders.CorHeader.Flags & (CorFlags.ILLibrary | CorFlags.ILOnly)) == 0)
+                    throw new NotSupportedException($"Error: managed C++ is not supported: '{filePath}'");
```

Comment:
```suggestion
                    throw new NotSupportedException($"Error: C++/CLI is not supported: '{filePath}'");
```
C++/CLI is the official name

Reply:
Just copy-pasted what crossgen2 prints, but I don't particularly like the message.